### ISSUE TITLE
Documentation - MdBook tracking issue

### ISSUE THREAD

**Issue by TimonPost** [djc/quinn#865]

**Comment by Ralith** [comment]
A couple quick notes:

> easy-rsa for a DIY certificate authority

This should be dropped in favor of rcgen.

> Streams - Application datagrams

We shouldn't describe these as a type of stream, though we should definitely discuss them and compare/contrast to unidirectional streams.

**Comment by TimonPost** [comment]
What would you propose as a topic name instead of streams?

**Comment by djc** [comment]
Maybe something like "Data transfer"?

(Note that your capitalization is pretty inconsistent here. I would prefer that we consistently just use "Single capital letter per title" rather than "Capitals For Every Word".)

**Comment by TimonPost** [comment]
Good point to keep in mind when realizing, those titles are still a draft and are mostly the copy of @Ralith proposal of the other issue. I'lll fix it directly to make it more representative to how the book will look like.

**Comment by TimonPost** [comment]
Locally I also have a branch ready for the Networking Introduction. I will be working on the Quinn section now. When this PR quinn-rs/quinn#868 is merged I will create a pull request for the Network Introduction chapter.
